study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/activeTaskScheduled.jsp

```diff
@@ -433,9 +440,9 @@
     <div class="mt-md">
       <div class="gray-xs-f mb-sm">Time(s) of the day for daily occurrence
         <span class="requiredStar"> *</span>
-        <span class="ml-xs sprites_v3 filled-tooltip" data-toggle="tooltip"
+        <span class="ml-xs sprites_v3 filled-tooltip Selectedtooltip" data-toggle="tooltip"
               data-placement="bottom" id="helpNote"
-              title="All date/time selections are based on server timezone."></span>
+              title="Selected dates and times will work as per the mobile device time. Selections of dates or times in the past (as per the server timezone which is ${server_timezone}) is not allowed."></span>
```

Comment:
Have tech writer help - same applies to all other instances

Reply:
Fixed